src/lib.rs

```diff
@@ -1,704 +1,18 @@
-#[macro_use]
-pub mod macros;
-
-use clear_on_drop::clear::Clear;
-use core::ops::{Add, Div, Mul, Sub};
-use curve25519_dalek::constants::{RISTRETTO_BASEPOINT_POINT, RISTRETTO_BASEPOINT_COMPRESSED};
-use curve25519_dalek::ristretto::{CompressedRistretto, RistrettoPoint};
-use curve25519_dalek::scalar::Scalar;
-use rand_core::{RngCore, CryptoRng, OsRng, };
-use serde::{Deserialize, Serialize};
-use sha2::{Digest, Sha512};
+#![feature(external_doc)]
```

Comment:
This feature is part of unstable and the only reason we need to use nightly. It seems to me that using the unstable  channel just because of documentation is not good reason enough.